src/app/signup/components/SignUpAccount.js

```diff
@@ -27,6 +29,14 @@ export function SignUpContainer({ children, sx }) {
 }
 
 export default function SignUpAccount({ goToNextStep }) {
+
+  const mutation = useMutation((value) => signupStep1({ email: value.email, password1: value.password1, password2: value.password2 }));
```

Comment:
Why not useMutation(signupStep1)?

Reply:
in this case, we will send termsConditionsAccepted too, but, I think, that doesn't matter